metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -608,6 +608,10 @@ def build_aggregated_measures(
         if len(output_nodes) == 1:
             return output_nodes[0]
         else:
+            if len(queried_linkable_specs.as_tuple) == 0:
+                raise NotImplementedError(
+                    "Querying metrics that requires a join with no dimensions is not yet supported."
+                )
```

Comment:
Let's give the user a little bit more context here. Something like this, perhaps, but with proper formatting.

```suggestion
                raise NotImplementedError(
                    "Querying metrics that require a join with no dimensions is not yet supported. This can happen with a metric that pulls measures from different data sources, or a metric that pulls measures with different constraints or non-additive dimension specifications."
                )
```